flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
this is disappointing.

Reply:
This project currently only supports X86_64 builds.
See https://github.com/chen08209/FlClash/blob/8d4931c09365b47b2d799f3cbf705b9c697609c7/setup.dart#L95-L99

Reply:
nonetheless, it's disappointing.